lib/api/concrexit_api_repository.dart

```diff
@@ -1119,4 +1119,53 @@ class ConcrexitApiRepository implements ApiRepository {
       }
     });
   }
+
+  @override
+  Future<ListResponse<ListGroup>> getGroups({
+    int? limit,
+    int? offset,
+    MemberGroupType? type,
+    DateTime? start,
+    DateTime? end,
+    String? search,
+  }) async {
+    const memberGroupTypeMap = {
+      MemberGroupType.committee: 'committee',
+      MemberGroupType.society: 'society',
+      MemberGroupType.board: 'board',
+    };
+
+    final uri = _baseUri.replace(
+      path: '$_basePath/activemembers/groups/',
+      queryParameters: {
+        if (limit != null) 'limit': limit.toString(),
+        if (offset != null) 'offset': offset.toString(),
+        if (type != null) 'type': memberGroupTypeMap[type],
```

Comment:
This should be some kind of `to_string` method or something.

Reply:
no, `type` should have such a method

Reply:
type could be extended to provide this (dart has some nice syntax for it these days), but since the string here js strictly used in the api, and not necessarily universal, it also makes sense to transform it here like this, so I really don't care